src/Arcus.EventGrid.Testing/Infrastructure/Hosts/EventConsumerHost.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using Polly;
+
+namespace Arcus.EventGrid.Testing.Infrastructure.Hosts
+{
+    public class EventConsumerHost
```

Comment:
Maybe some documentation for this class?
Otherwise nice and clean!

Reply:
Good point :+1: